src/applications/renderer/src/components/standalone/fetch-layers-hook.js

```diff
@@ -3,7 +3,9 @@ import { useState, useEffect, useMemo } from "react";
 const useLayerData = (adapter, layerId, isMap) => {
   const [layerData, setData] = useState(null);
 
-  const [isLoadingLayers, setIsLoading] = useState(false);
+  // We set the default state to loading so that this hook never returns null on initialization
+  const [isLoadingLayers, setIsLoading] = useState(true);
```

Comment:
This change is enough to fix the issue, but as a safety net, I've also added a check in the `component.js` file (see above).